torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1790,6 +1810,7 @@ def hint():
                                 return "Did you really mean to mark this dimension as dynamic?"
 
                         record_constraint_violation(lambda: (
+                            constraint,
```

Comment:
instead of putting it in the lambda, why not have it as an argument to the function directly?

Reply:
Cause I want to record it, but don't want to make a whole new structure for it